winrm4j/src/main/java/io/cloudsoft/winrm4j/winrm/WinRmTool.java

```diff
@@ -49,10 +134,14 @@ public WinRmToolResponse executeCommand(List<String> commands) {
      * @since 0.2
      */
     public WinRmToolResponse executeCommand(String command) {
-        Builder builder = WinRmClient.builder(getEndpointUrl());
+        WinRmClient.Builder builder = WinRmClient.builder(address, authenticationScheme);
         if (username != null && password != null) {
             builder.credentials(username, password);
         }
+        if (disableCertificatesChecks) {
+            LOG.info("Disabled check for https connections " + this);
```

Comment:
Also a `debug` would be more appropriate, at least until we support certificate white-listing.